docs/table-migration.md

```diff
@@ -0,0 +1,89 @@
+
+In-place migration preserves the existing data files while incorporating Iceberg metadata on top of them.
+This method is not only faster but also eliminates the need for data duplication, making it more appropriate for production environments.
+
+![In-Place Migration](../../../img/iceberg-in-place-migration.png)
+
+Apache Iceberg primarily supports the in-place migration approach, which includes three important actions: `Snapshot Table`, `Migrate Table`, and `Add Files`.
+
+## Snapshot Table
+The Snapshot Table action creates a new iceberg table with the same schema and partitioning as the source table, leaving the source table unchanged during and after the action.
+
+- Create a new Iceberg table with the same metadata (schema, partition spec, etc.) as the source table
+
+![Snapshot Table Step 1](../../../img/iceberg-snapshotaction-step1.png)
+
+- Commit all data files across all partitions to the new Iceberg table. The source table remains unchanged.
+
+![Snapshot Table Step 2](../../../img/iceberg-snapshotaction-step2.png)
+## Migrate Table
+The Migrate Table action also creates a new Iceberg table with the same schema and partitioning as the source table. However, during the action execution, it locks and drops the source table from the catalog.
+Consequently, Migrate Table requires all modifications working on the source table to be stopped before the action is performed.
+
+Stop all writers interacting with the source table. Readers that also supports Iceberg may continue reading.
+
+![Migrate Table Step 1](../../../img/iceberg-migrateaction-step1.png)
+
+- Create a new Iceberg table with the same identifier and metadata (schema, partition spec, etc.) as the source table. Rename the source table for a backup in case of failure and rollback.
+
+![Migrate Table Step 2](../../../img/iceberg-migrateaction-step2.png)
+
+- Commit all data files across all partitions to the new Iceberg table. Drop the source table.
+
+![Migrate Table Step 3](../../../img/iceberg-migrateaction-step3.png)
+## Add Files
+After the initial step (either Snapshot Table or Migrate Table), it is common to find some data files that have not been migrated. These files often originate from concurrent writers who continue writing to the source table during or after the migration process.
+In practice, these files can be new data files in Hive tables or new snapshots (versions) of Delta Lake tables. The Add Files action is essential for incorporating these files into the Iceberg table.
+
+## In-Place Migration Completion
+Once all data files have been migrated and there are no more concurrent writers writing to the source table, the migration process is complete.
+Readers and writers can now switch to the new Iceberg table for their operations.
+
+# Migrating From Different Table Formats
+## From Hive to Iceberg
```

Comment:
based on our discussion, since Delta is in separated page, Hive can also be on a separated page

Reply:
I added a new page called `Hive-Migration` and make it reside in the `Migration` subgroup